packages/shared/src/components/MainLayout.tsx

```diff
@@ -128,7 +130,7 @@ export default function MainLayout({
           hasBanner ? 'laptop:pt-22' : 'laptop:pt-14',
         )}
       >
-        {!showOnlyLogo && sidebarRendered !== null && (
+        {showSidebar && !showOnlyLogo && sidebarRendered !== null && (
```

Comment:
Are you sure this is right?
Not 100% sure this should fire as it has the mobile hidden sidebar?

Reply:
Nice catch! Just tested it now, the sidebar is now available on mobile and only hidden on desktop.

Reply:
Good point, I agree, better to ask them as this is a new behavior.

Reply:
Tsahi has confirmed, this is indeed a new behavior where we show header buttons but sidebar is hidden 🚢